client.go

```diff
@@ -221,7 +221,8 @@ func (c *Client) fullURL(suffix string, args ...any) string {
 		baseURL = strings.TrimRight(baseURL, "/")
 		// if suffix is /models change to {endpoint}/openai/models?api-version=2022-12-01
 		// https://learn.microsoft.com/en-us/rest/api/cognitiveservices/azureopenaistable/models/list?tabs=HTTP
-		if strings.Contains(suffix, "/models") || strings.Contains(suffix, "/assistants") {
+		if strings.Contains(suffix, "/models") || strings.Contains(suffix, "/assistants") ||
```

Comment:
Thank you for the PR!

Could we refactor this please?

Reply:
That would be cool https://freshman.tech/snippets/go/check-if-slice-contains-element/
But I think it makes sense to postpone to keep more go versions supported

Reply:
@sashabaranov are you on-board with using the golang.org/x/exp/slices package to support 1.18+ ?

Reply:
@coggsflod let's just add a small helper functions without any extra dependencies

Reply:
@sashabaranov actually all these [options](https://freshman.tech/snippets/go/check-if-slice-contains-element/) perform an exact match check rather than the strings.Contains which checks whether substr is within passed string.

Happy to create a containsSubstr helper if that works?